src/pages/Profile.jsx

```diff
@@ -69,6 +131,13 @@ function Profile() {
           >
             Back
           </Button>
+          {visible && (<Button
```

Comment:
move the save button below or above edit profile pic(perhaps above is better ux)

Reply:
done